src/rt/alloca.d

```diff
@@ -11,6 +11,8 @@
 
 module rt.alloca;
 
+version (LDC) {} else:
+
```

Comment:
This file is excluded from compilation in CMake, but make that clear in the file as well by versioning it out completely.